CHANGES_NEXT_RELEASE

```diff
@@ -11,4 +11,5 @@
 - Group and redefine error codes.
 - Add execution mode without Access Control authorization for the PEP Proxy (#173)
 - Check the service and subservice headers for content (#176)
-- Fix the Keystone user authentication error was raised as a connection one (#174)
\ No newline at end of file
+- Fix the Keystone user authentication error was raised as a connection one (#174)
+- Add a guard in the release script to abort if there are unstagged git changes (#181)
```

Comment:
Probably in this case it is not needed, as release.sh is not actual part of the PEP code but a "helping tool" in the repo. However, it doesn't hurt to have the line in the file, so let's keep as it is now.

NTC